pouchdb-mapreduce/index.d.ts

```diff
@@ -17,6 +78,14 @@ declare namespace PouchDB {
          */
         viewCleanup(callback: PouchDB.Core.Callback<any,void>): void;
         viewCleanup(): Promise<void>;
+
+        /**
+         * Invoke a map/reduce function, which allows you to perform more complex queries
+         * on PouchDB than what you get with allDocs().
+         */
+        query(fun: string | Filter | Function, opts: Query.Options, callback: (err: Core.Error, res: Query.Response<Content>) => void): void;
+        query(fun: string | Filter | Function, callback: (err: Core.Error, res: Query.Response<Content>) => void): void;
```

Comment:
Please make doc info for each method. You can also double docs for viewCleanup function above. That would be all, thanks .) GJ